Lib/configparser.py

```diff
@@ -561,36 +561,31 @@ def __init__(self):
         self.errors = list()
 
 
-class _Line(str):
+class _LineParser:
 
-    def __new__(cls, val, *args, **kwargs):
-        return super().__new__(cls, val)
-
-    def __init__(self, val, prefixes):
-        self.prefixes = prefixes
-
-    @functools.cached_property
-    def clean(self):
-        return self._strip_full() and self._strip_inline()
+    def __init__(self, comments):
+        self.comments = comments
 
     @property
-    def has_comments(self):
-        return self.strip() != self.clean
+    def value(self):
+        return self._value
 
-    def _strip_inline(self):
-        """
-        Search for the earliest prefix at the beginning of the line or following a space.
-        """
```

Comment:
This comment seems to have been lost.

Reply:
moved to 659 since it explains the regular expression